client/kaia_client.go

```diff
@@ -581,6 +590,12 @@ func toSendTxArgs(msg api.SendTxArgs) interface{} {
 	if msg.Price != nil {
 		arg["gasPrice"] = (*hexutil.Big)(msg.Price)
 	}
+	if msg.MaxFeePerGas != nil {
```

Comment:
`MaxFeePerGas` and `MaxPriorityFeePerGas` cannot be passed through `client.SendUnsignedTransaction`. 
https://github.com/kaiachain/kaia/blob/f426d075625bff5b3281ec0e69fc4e75a3be3756/client/kaia_client.go#L520-L532

Could you make the function able to pass those fields also?

Reply:
Fixed in https://github.com/kaiachain/kaia/pull/29/commits/cbdca4769b5253b583410032e6285a80ea101a7e. I let caller to fill up the SendTxArgs, like how client.SendTransaction just receives Transaction struct.